stdlib/number.gr

```diff
@@ -351,6 +430,14 @@ provide let isRational = (x: Number) => {
  * @param x: The number to check
  * @returns `true` if the value is finite or `false` otherwise
  *
+ * @example Number.isFinite(1/2)
+ * @example Number.isFinite(0.5)
+ * @example Number.isFinite(1.0)
+ * @example Number.isFinite(1)
```

Comment:
And these?

Reply:
Or is this our style? Just feels a little weird since it's not an assertion.

Reply:
I haven't been paying that close of attention. I know jake was doing assert in some cases.

Reply:
We have been using `assert` semantics for single line matches throughout the stdlib.

Reply:
@spotandjake why don't these contain `assert` then?